Civi/RemoteTools/Form/FormSpec/Field/RadiosField.php

```diff
@@ -0,0 +1,39 @@
+<?php
+/*
+ * Copyright (C) 2024 SYSTOPIA GmbH
+ *
+ *  This program is free software: you can redistribute it and/or modify
+ *  it under the terms of the GNU Affero General Public License as published by
+ *  the Free Software Foundation in version 3.
+ *
+ *  This program is distributed in the hope that it will be useful,
+ *  but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ *  GNU Affero General Public License for more details.
+ *
+ *  You should have received a copy of the GNU Affero General Public License
+ *  along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+declare(strict_types = 1);
+
+namespace Civi\RemoteTools\Form\FormSpec\Field;
+
+/**
+ * @extends AbstractOptionField<int|string>
+ *
+ * @codeCoverageIgnore
+ *
+ * @api
+ */
+class RadiosField extends AbstractOptionField {
+
+  public function getDataType(): string {
+    return 'scalar';
+  }
+
+  public function getInputType(): string {
+    return 'radio';
```

Comment:
Just checking … so *JSON Forms* calls multiple radio buttons `radio` which is why this doesn't match the class name now …?

Reply:
As this is independent of JSON Forms it makes sense to match the class name so I changed that.